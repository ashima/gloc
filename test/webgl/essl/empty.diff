--- empty.true	2011-11-14 16:22:51.000000000 -0800
+++ empty.out	2011-11-17 16:38:56.000000000 -0800
@@ -1 +1,10 @@
-error
+void main() {
+  ;
+  ;
+  {}
+  {;}
+  {;{};}
+  {;{;};}
+
+  gl_FragColor = vec4(0.);
+}
