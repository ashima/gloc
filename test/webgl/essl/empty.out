#line 1 0/* <stdin>#u=0 */
void main() {
  ;
  ;
  {}
  {;}
  {;{};}
  {;{;};}

  gl_FragColor = vec4(0.);
}

